### PR TITLE
Update helm charts with deployment support

### PR DESCRIPTION
**Pull request by Yuni-sa**
for #40 and #87.

### CODE REVIEW

**Comment by yonahd** [comment]
Can you add a ServiceMonitor to the chart

File: charts/kor/values.yaml [hunk 1]
```diff
@@ -14,6 +14,45 @@ cronJob:
...
+prometheusExporter:
+  enabled: true
+  name: kor-exporter
+  # time in minutes, default is 10 minutes
+  exporterInterval: ""
+  deployment:
+    image:
+      repository: yonahdissen/kor
+      tag: latest
+    command: kor exporter
```
Comment (patricktalmeida) [inline charts/kor/values.yaml:26]: Wouldn't it be better if this command specified the destination where it's exporting to? I mean maybe in the future some other feature could try to export somewhere else other then to Prometheus.
What do you think of having something like:

`kor exporter --prometheus`
or
 `kor prom-exporter`

**Comment by yonahd** [comment]
LGTM!
Waiting for https://github.com/yonahd/kor/pull/87 before merging

**Review by yonahd — APPROVED** [review]
LGTM

**Pull request merged**